cpp/oneapi/dal/algo/linear_regression/backend/gpu/infer_kernel_norm_eq_dpc.cpp

```diff
@@ -109,7 +109,7 @@ static infer_result<Task> call_dal_kernel(const context_gpu& ctx,
     ONEDAL_ASSERT((f_count + 1) == betas.get_column_count());
 
     const auto resps_size = check_mul_overflow(s_count, r_count);
-    auto resps_arr = array<Float>::empty(queue, resps_size);
```

Comment:
I suggest not to mix enhancement and bug fixing in one PR

Reply:
Agreed. But in this case these fixes are pretty small